Tests/phpunit/ExportConfigTest.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\DataLoader\FunctionalTests;
+
+use Keboola\DataLoader\ExportConfig;
+use Keboola\DataLoader\TransformationConfig;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Config\Definition\Exception\InvalidConfigurationException;
+use Symfony\Component\Config\Definition\Processor;
+
+class ExportConfigTest extends TestCase
+{
+    /**
+     * @dataProvider configurationProvider
+     * @param array $input
+     * @param array $expected
+     */
+    public function testSuccess(array $input, array $expected): void
+    {
+        $processor = new Processor();
+        $outData = $processor->processConfiguration(new ExportConfig(), ['configuration' => $input]);
+        self::assertEquals($expected, $outData);
+    }
+
+    public function configurationProvider(): array
+    {
+        return [
+            [
+                [],
+                [],
+            ],
+            [
```

Comment:
We can give these arrays keys and they are used as test names.  A handy tip that TF showed me:)